### PR TITLE
Add email credentials to deployment.properties

### PR DESCRIPTION
**Pull request by madushadhanushka**
$subject

### CODE REVIEW

File: main.groovy [hunk 1]
```diff
@@ -55,7 +55,8 @@ stages {
                 string(credentialsId: 'WUM_PASSWORD', variable: 'wumPassword'),
                 string(credentialsId: 'DEPLOYMENT_DB_PASSWORD', variable: 'dbPassword'),
                 string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 's3accessKey'),
-                string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 's3secretKey')])
+                string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 's3secretKey'),
+                string(credentialsId: 'TESTGRID_EMAIL_PASSWORD', variable: 'TESTGRID_EMAIL_PASSWORD')])
```
Comment (janethavi) [inline main.groovy:59]: shall we rename the variable value to something like `testgridEmailPassword`? Also, is this specific to APIM?

File: scripts/deployment-handler.sh [hunk 2]
```diff
@@ -95,6 +95,7 @@ function addCommonVariables(){
     writeCommonVariables "ProductVersion" "ProductVersion" true
     writeCommonVariables "S3AccessKeyID" "s3accessKey" true
     writeCommonVariables "S3SecretAccessKey" "s3secretKey" true
+    writeCommonVariables "TESTGRID_EMAIL_PASSWORD" "TESTGRID_EMAIL_PASSWORD" true
```
Comment (janethavi) [inline scripts/deployment-handler.sh:98]: if we are doing the 1st mentioned change, then the 2nd parameter should be changed to `testgridEmailPassword` too.

**Review by janethavi — CHANGES_REQUESTED** [review]

File: main.groovy (hunk 1 above)
Comment (madushadhanushka) [reply main.groovy:59]: currently I think this is for APIM

Comment (madushadhanushka) [reply main.groovy]: As this is email credentials, any product can use it send emails.

**Review by janethavi — APPROVED** [review]

**Pull request merged**